base/server/python/pki/server/__init__.py

```diff
@@ -578,6 +578,60 @@ def cert_del(self, cert_tag, remove_key=False):
         finally:
             nssdb.close()
 
+    def cert_import_nssdb(self, cert_tag, cert_file=None):
```

Comment:
Can we just call it cert_import()?

Reply:
I interpret `cert_import()` as _import into PKI system_ (ie) updates all required files like `CS.cfg` and `nssdb`. But, this method actually just _imports into the PKI system's NSS db_

We have a separate method called `cert_update_config()` defined under `PKIInstance` to actually update the `CS.cfg`.

Reply:
OK, how about nssdb_import_cert()? Usually we put the parameter (i.e. the object of the action) at the end so that the method can be grouped together with other similar (i.e. nssdb_*) methods.